test/acceptance_with_go_client/autoschema_test.go

```diff
@@ -77,13 +75,73 @@ func TestAutoschemaCasingProps(t *testing.T) {
 	}
 	for _, tt := range cases {
 		t.Run(tt.prop1+" "+tt.prop2, func(t *testing.T) {
+			c.Schema().ClassDeleter().WithClassName(className).Do(ctx)
+			creator := c.Data().Creator()
+			_, err := creator.WithClassName(className).Do(ctx)
+			require.Nil(t, err)
+
 			creator1 := c.Data().Creator()
-			_, err := creator1.WithClassName(className).WithProperties(map[string]string{tt.prop1: "something"}).Do(ctx)
+			_, err = creator1.WithClassName(className).WithProperties(map[string]string{tt.prop1: "something"}).Do(ctx)
 			require.Nil(t, err)
 
 			creator2 := c.Data().Creator()
-			_, err = creator2.WithClassName(className).WithProperties(map[string]string{tt.prop2: "overwrite value"}).Do(ctx)
+			_, err = creator2.WithClassName(className).WithProperties(map[string]string{tt.prop2: "other value"}).Do(ctx)
+			require.Nil(t, err)
+
+			// three objects should have been added
+			result, err := c.GraphQL().Aggregate().WithClassName(className).WithFields(graphql.Field{
+				Name: "meta", Fields: []graphql.Field{
+					{Name: "count"},
+				},
+			}).Do(ctx)
+			require.Nil(t, err)
+			require.Equal(t, result.Data["Aggregate"].(map[string]interface{})[className].([]interface{})[0].(map[string]interface{})["meta"].(map[string]interface{})["count"], 3.)
+
+			require.Nil(t, c.Schema().ClassDeleter().WithClassName(className).Do(ctx))
+		})
+	}
+}
+
+func TestAutoschemaCasingUpdateProps(t *testing.T) {
+	ctx := context.Background()
+	c := client.New(client.Config{Scheme: "http", Host: "localhost:8080"})
+
+	objId := "67b79643-cf8b-4b22-b206-6e63dbb4e57a"
+	upperPropName := "SomeProp"
+	lowerPropName := "someProp"
+	cases := []struct {
+		prop1 string
+		prop2 string
```

Comment:
what is `prop2` for? seems to be used just as test name

Reply:
whoopts, that should have been the name for the update, fixed